tests/test_dimming_actuator.py

```diff
@@ -0,0 +1,138 @@
+"""Test class to test the DimmingActuator device."""
+
+from unittest.mock import AsyncMock
+
+import pytest
+
+from abbfreeathome.api import FreeAtHomeApi
+from abbfreeathome.devices.dimming_actuator import DimmingActuator
+
+
+@pytest.fixture
+def mock_api():
+    """Create a mock api function."""
+    return AsyncMock(spec=FreeAtHomeApi)
+
+
+@pytest.fixture
+def dimming_actuator(mock_api):
+    """Set up the dimming instance for testing the DimmingActuator device."""
+    inputs = {
+        "idp0000": {"pairingID": 1, "value": "0"},
+        "idp0002": {"pairingID": 17, "value": "50"},
+    }
+    outputs = {
+        "odp0000": {"pairingID": 256, "value": "0"},
+        "odp0001": {"pairingID": 272, "value": "50"},
+    }
```

Comment:
Add an additional output that exists in a dimmer actuator that doesn't affect the state.

```suggestion
       outputs = {
        "odp0000": {"pairingID": 256, "value": "0"},
        "odp0001": {"pairingID": 272, "value": "50"},
        "odp0003": {"pairingID": 257, "value": "0"},
    }
```